### ISSUE TITLE
Añadir el objeto de un móvil con utilidades

### ISSUE THREAD

**Issue by MiguelDreamer** [ComunidadAylas/khron-plugins#19]
Con el ítem de un móvil (modelo y textura añadidos mediante el resource pack del servidor), los jugadores pueden obtener mucha información y realizar determinadas acciones. El móvil, de marca Red Energy y con un precio de 10.000 Khrones, tendrá una interfaz con la que el jugador puede activar recibir cierta información en la barra de acción (¿o barra del jefe?) mientras tenga el móvil en el inventario (la información cambia a otra cada 5 segundos dependiendo lo que tenga activado):

(En paréntesis, mensajes de ejemplo)

_- Hora y fecha actual (Hora: 12:34:56, fecha: lunes, 22/06/2020)
- Coordenadas (Coordenadas: X=20, Y=64, Z=30, mirando al oeste)
- Clima y temperatura (Clima: soleado, temperatura: 32º)
- Velocidad actual (8 km/h)_

Aplicaciones integradas en el móvil, de forma pasiva y activa:

_- Integración del comando tell/msg, que notifica con un sonido al recibir un mensaje privado_
- Pedir determinada comida personalizada del resource pack, se cobre al jugador y le llegue en 3-5 minutos.
- Aplicación de experiencia: ranking, que muestra el top 10 de jugadores con más experiencia, posición, que muestra los 5 jugadores por encima de ti y los 5 por debajo y perfil, que muestra tu experiencia y nivel actual y cuánto te queda para subir al siguiente.
- Aplicación del banco: al pulsarla, en el chat saldrá tu dinero actual.

Las siguientes "aplicaciones" o funciones se tienen que comprar en la tienda de aplicaciones y no vienen por defecto en el móvil:

- Información para activar de menas de minerales a menos de 16 bloques de distancia (¡Hay minerales cerca!). Precio: 50.000 Khrones.
- Información para activar de mobs hostiles a menos de 32 bloques de distancia (¡Hay enemigos cerca!). Precio: 5.000 Khrones.
- Información para activar de tesoros del mar a menos de 24 bloques de distancia (¡Hay un tesoro cerca!). Precio: 25.000 Khrones.
- Cambiar el color del nombre del jugador en el chat. Precio: 20.000 Khrones
- Poder ingresar y sacar dinero de la cuenta bancaria, independientemente de donde estés y en cualquier momento. Precio: 100.000 Khrones.

La utilidad y funciones del móvil están sujetos a cambios en el desarrollo. Lo escrito en cursiva puede ser la primera versión del móvil, y todo lo demás en futuras actualizaciones. Por la complejidad de varias funciones, puede que se requiera de mucho trabajo y esfuerzo. Estoy encantado de que hablemos sobre este tema y tengamos nuevas ideas y discutamos sobre este futuro posible plugin.

**Comment by AlexTMjugador** [comment]
Las ideas están bastante bien. He cambiado el hito de UtilidadesVarias a uno nuevo para el plugin Telefonos, por las razones que comenté en #16. Del móvil básico, quitaría la información de velocidad actual no porque sea difícil de calcular, sino porque tanto la barra de acción como la de jefe tienen un ancho limitado (especialmente si Minecraft no se juega a pantalla completa) y no admiten varias líneas de texto, por lo que los mensajes que ahí se muestren deben de ser cortos y relevantes.

**Comment by MiguelDreamer** [comment]
La idea es que la información fuese rotando cada 5 segundos (es decir, que luego de mostrar la fecha y la hora, mostrara las coordenadas y la localización, como escribí las cosas en orden). Aunque si te parece bien, se puede poner toda la información de forma muy compacta en la misma frase.

**Comment by AlexTMjugador** [comment]
Soy más partidario de mostrar todo a la vez si es posible, pero es algo que se puede cambiar si no es posible lo primero.

**Comment by MiguelDreamer** [comment]
Acabo de hacer una prueba en el juego y se vería así en la barra de acción, ¿qué te parece?
![2020-06-22_19 06 53](https://user-images.githubusercontent.com/31966940/85316044-5ae48b00-b4bc-11ea-8540-c3d134c67908.png)

**Comment by AlexTMjugador** [comment]
No está mal, pero el tamaño de la captura me hace pensar que estás usando una resolución de 1920x1080 píxeles. ¿Cómo se ve en otras resoluciones, como la predeterminada del modo ventana de Minecraft, 856x482? ¿Y con otras escala de la interfaz, que puedes cambiar en las opciones gráficas?

**Comment by MiguelDreamer** [comment]
Efectivamente, la captura de antes es en 1920x1080 píxeles. Lo bueno de Minecraft es que con la opción de "Tamaño de interfaz" todo se ajusta y cabe a la perfección. Esta captura tiene de resolución 856x482, con el tamaño de la interfaz en 1.
![2020-06-22_19 45 41](https://user-images.githubusercontent.com/31966940/85318953-21624e80-b4c1-11ea-837e-72c81cdd2027.png)

**Comment by AlexTMjugador** [comment]
Perfecto 👍

**Comment by MiguelDreamer** [comment]
Viendo las últimas adiciones en TiempoReal, estoy seguro que la información quedaría visualmente mucho mejor con los nuevos iconos, en vez de como la última foto que pasé (¡me encanta lo que hiciste!). Además, creo que parte del código ya lo has hecho, y el camino para hacer esto posible está más cerca que nunca.
